.github/actions-rs/grcov.yml

```diff
@@ -1,8 +0,0 @@
-ignore-not-existing: true
```

Comment:
Was this a dead file?

Reply:
Yes it hasnt been used for a long time now